lib/net/peer.js

```diff
@@ -285,7 +314,13 @@ class Peer extends EventEmitter {
     this.address.services = 0;
     this.outbound = false;
     this.connected = false;
-    this.brontide.accept(socket, this.identityKey);
+
+    if (this.encrypt) {
+      this.brontide.accept(socket, this.identityKey);
+    } else {
+      this.connected = true;
+      this.time = Date.now();
+    }
```

Comment:
This `else` block comes directly from the `bcoin` codepath

Reply:
Would it make sense to build an interface that matches the `BrontideStream` for the regular `socket`?

Reply:
Updating the above comment. The `STANDARD` case is directly from the `bcoin` codepath.